src/makeStyles.tsx

```diff
@@ -56,7 +56,7 @@ export function createMakeStyles<Theme>(params: { useTheme: () => Theme }) {
 
             const outerCounter = getCounter();
 
-            return function useStyles(params: Params) {
+            return function useStyles(params?: Params) {
```

Comment:
In recent TypeScript a function of type:
```typescript
declare function f(x: void);
```

Can be invoked like that:

```typescript
f();
```

[See compile time unit tests](https://github.com/garronej/tss-react/blob/63715700e16246ab3027ada4e712c738de3e9e2a/src/test/types/makeStyles.tsx#L44)